OrbitVulkanLayerClientGgp/LayerTimes.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_VULKAN_LAYER_CLIENT_GGP_LAYER_TIMES_H_
+#define ORBIT_VULKAN_LAYER_CLIENT_GGP_LAYER_TIMES_H_
+
+#include "absl/time/time.h"
+
+struct LayerTimes {
```

Comment:
If you don't plan to use this struct anywhere else I would make the two Time's member of LayerLogic.  If you plan to pass these around separately later please ignore this.

Reply:
That makes sense, I am not planning to use it anywhere else so I removed it. Thanks!